### ISSUE TITLE
Assertion fails: src/bzlafp.cpp:721

### ISSUE THREAD

**Issue by merlinsun** [bitwuzla/bitwuzla#48]
Greetings,
For this instance, bitwuzla https://github.com/bitwuzla/bitwuzla/commit/1230d80a5275ddf525da9c218c2bcd29c3130e49  debug build
```
$ bitwuzla small.smt2       
bitwuzla: /root/smt/bitwuzla/src/bzlafp.cpp:721: static void BzlaFPTraits::postcondition(const bool&): Assertion `p' failed.
[bitwuzla>main] CAUGHT SIGNAL 6
unknown
Aborted (core dumped)
$ cat small.smt2  
(assert ((fp.sqrt RNE (fp (_ bv0 1) (_ bv0 3) (_ bv1 3)))))
```

**Comment by merlinsun** [comment]
It appear to be related to #47.

**Comment by mpreiner** [comment]
Duplicate of https://github.com/bitwuzla/bitwuzla/issues/47

**Comment by mpreiner** [comment]
Fixed in new 0.1.0 release.
